sdcm/nemesis.py

```diff
@@ -4225,7 +4225,7 @@ def disrupt_add_remove_mv(self):
             InfoEvent(message=f'Create a materialized-view for table {ks_name}.{base_table_name}').publish()
             try:
                 with EventsFilter(event_class=DatabaseLogEvent,
-                                  regex='.*view - Error applying view update.*',
+                                  regex='.*Error applying view update.*',
```

Comment:
The error is:
```
!ERR | scylla[13223]:  [shard 12] view - Error applying view update to 10.12.8.34 (view: mview.users_view, base token: -9214854270618842917, view token: -8922807318382248831): exceptions::unavailable_exception (Cannot achieve consistency level for cl ONE. Requires 1, alive 0)
```

May be set regex: 
```
'.*Error applying view update.*::unavailable_exception'
```

Reply:
does it need to have a `*` in the end of the of the line too?

Reply:
yes. But I think the suggested filter is good enough and doesn't have any risk, so better keep-it-simple.

Reply:
what @juliayakovlev suggest is to search for `error applying view update`, but with specific exception called `unavailable_exception`
what are other types of exception we could see in this case, @yarongilor ?

Reply:
I don't know, there are currently 2 types of them, so i don't want to take a risk, discovering more of them failing more future tests.

Reply:
i don't want to take a risk and ignore real issues, because we are filtering too broad cases
@fruch @roydahan WDYT?